support.md

```diff
@@ -0,0 +1,8 @@
+# Support
```

Comment:
Hey @leahyc, is this documentation still accurate? Do you have a more up to date version?